src/main/java/com/jpn/lang/controller/EmployeeControllerApi.java

```diff
@@ -3,25 +3,85 @@
 import com.jpn.lang.model.request.EmployeePostRequest;
 import com.jpn.lang.model.request.EmployeePutRequest;
 import com.jpn.lang.model.response.EmployeeResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestParam;
-
 import java.util.ArrayList;
 
 public interface EmployeeControllerApi {
+
+    @Operation(
+            tags = "ADD Employees",
+            description = "Add Employees",
+            responses = {
+                    @ApiResponse(
+                            description = "Success",
+                            responseCode = "200"
+                    ),
+                    @ApiResponse(
+                            description = "Data Not Found",
```

Comment:
Will you get this error in case of post request?

Reply:
i do not get this error on browser.
i get only empty array.

Reply:
Thats what I am saying you will not get this error in case of post API. Because you are not modifying or deleting existing object. instead you are creating new object. So you can remove Data not found response from post API.